packages/world/src/interfaces/IWorldRegistrationSystem.sol

```diff
@@ -23,9 +23,9 @@ interface IWorldRegistrationSystem {
 
   function registerRootFunctionSelector(
```

Comment:
Should this be just “register root function”?

Reply:
i personally find it more intuitive to keep the selector, since what happens is that a new function selector is added to the world contract (it's debatable, but there isn't really a new function added to the world, the function is part of the system and it was already possible to call it before a function selector was registered for it)

Reply:
but we are effectively adding a function to the world, regardless of how it’s implemented

I kinda like that “register function” communicates “world is mutable/extendable”

but happy to keep as is if you feel strongly

Reply:
In my mine we're just adding a "link" to an existing function by adding a new "function selector" to the World here. The World was extended when the system was registered (and it's even cheaper to call a function via the World's `call` function than going the indirection over the "function selector link" in the World)

Reply:
ohh I see, we're aliasing a function's name? I thought this is what system registration used to enable you to call the system methods via the world